%COMPILE WITH LuaLaTeX
\documentclass{gbcv}

% include PubMed ID info for publications
\newif\ifpm
\pmfalse
%\pmtrue

% include contribution explanations for RPT
\newif\ifrpt
\rptfalse
%\rpttrue

\newcolumntype{R}{>{\raggedleft\arraybackslash}p{0.295\textwidth}}
\setlength{\parindent}{0in}
\pagestyle{fancy}

\begin{document}\thispagestyle{empty}
\pagenumbering{gobble}
%
\begin{huge}
\bf{Gideon Bradburd}
\hfill\normalfont{\small{\textbf{\today}}}\
\end{huge}
\vspace{-0.25cm}
\\
%
\vspace{-0.7cm}
\rule{470pt}{0.4pt}
%\noindent\makebox[\linewidth]{\rule{18cm}{0.4pt}}
\vspace{0.3cm}
%
\\
3042 Biological Sciences Building \hfill email: \href{mailto:bradburd@umich.edu}{bradburd@umich.edu}\\
Department of Ecology \& Evolutionary Biology \hfill website: \href{http://genescape.org}{genescape.org}\\
%\hfill phone: 203-687-5224\\
University of Michigan \hfill software: \href{https://github.com/gbradburd}{github.com/gbradburd}\\

%\vspace{-0.25cm}
\section*{EDUCATION \& APPOINTMENTS}
\vspace{-0.6cm}
\rule{470pt}{0.4pt}
%
\begin{tabular}{p{0.25\textwidth} p{0.5\textwidth} p{0.175\textwidth}} 
\\
\rule{0pt}{4ex}\:\bf{Asst. Professor} & \it{University of Michigan} & \hfill 2022 - present\\
 & \hspace{0.5cm}Department of Ecology and Evolutionary Biology & \\
%
\rule{0pt}{4ex}
%
\bf{Asst. Professor} & \it{Michigan State University} & \hfill 2016 - 2022\\
 & \hspace{0.5cm}Department of Integrative Biology & \\
 & \hspace{0.5cm}Ecology, Evolution, and Behavior Program& \\
  & \hspace{0.5cm}fixed-term (2016-17); tenure-stream (2017-2022)& \\
%
\rule{0pt}{4ex}
%
\bf{Postdoc} & \it{University of California, Berkeley} & \hfill 2015 - 2016\\
 & \hspace{0.5cm}Museum of Vertebrate Zoology & \\
 & \hspace{0.5cm}Department of Env.\ Sci.\ Pol.\ and Mgmt& \\
 & \hspace{0.5cm}Advisors: Michael Nachman, Bree Rosenblum & \\ 
%
\rule{0pt}{4ex}
%
\bf{PhD} & \it{University of California, Davis}  & \hfill 2009 - 2015\\
 & \hspace{0.5cm}Population Biology Graduate Group & \\
 & \hspace{0.5cm}Department of Ecology and Evolutionary Biology & \\
 & \hspace{0.5cm}Advisors: Graham Coop, Brad Shaffer & \\ 
%
\rule{0pt}{4ex}
%
\textbf{Bachelor of Science} & \it{Yale University} & \hfill 2004 - 2008 \\
& \hspace{0.5cm}Ecology and Evolutionary Biology & \\
& \hspace{0.5cm}Honors Thesis Advisor: Tom Near & \\
\end{tabular}
%
\vspace{0.1cm}
\section*{LEAVES \& MODIFIED DUTIES \hfill}
\vspace{-0.6cm}
\rule{470pt}{0.4pt}
\begin{tabular}{>{\everypar{\hangindent1cm}}p{0.9\textwidth}p{0.05\textwidth}}
\hfill\\
\textbf{2020} \hspace{1cm}Second child born in Aug, 6 weeks parental leave, teaching release Spring Semester 2021\\
\textbf{2016} \hspace{1cm}First child born in November, 3 weeks parental leave, one-year tenure clock extension
\end{tabular}

\vspace{0.36cm}
\section*{PUBLICATIONS \hfill \normalfont{\small{*advisees (e.g., students/postdocs)}}}
\vspace{-0.6cm}
\rule{470pt}{0.4pt}
%
\vspace{-0.9cm}
%
\begin{longtable}{>{\everypar{\dohang}\dohang\raggedright\arraybackslash}p{0.95\textwidth}}
\hfill\\
\textbf{\underline{\smash{Preprints, In Review, and In Revision}}} \hfill\\
%
\rule{0pt}{6ex} 
\labbie{Hancock, ZB}, NE Adams, PL Wood, \bburd{GS Bradburd}.
\pubyear{2024}.
Asymmetric gene flow maintains an abundant range center.
\href{https://www.biorxiv.org/content/10.1101/2024.11.04.621918v2}{\underline{bioRxiv}}.
\ifrpt 
	\\[\pubspace em]
	\contribution{
		This paper is led by a postdoc mentee in my lab (lead author). 
		I am senior author. 
		I contributed to writing and idea development, and mentored on analyses.
		\\[\littlepubspace em]
	}
	\dohang
	\\\pagebreak
\else
\\[\littlepubspace em]
\fi

%
Buysse, SF, SG P\'erez, JR Puzey, A Garrison, \bburd{GS Bradburd}, 
CG Oakley, X Pic\'o, SJ Tonsor, EB Josephs, JK Conner.
\pubyear{2024}.
The roles of drift and selection on short stamen loss in 
\textit{Arabidopsis thaliana} along an elevational gradient in the Spanish Pyrenees.
\href{https://www.biorxiv.org/content/10.1101/2024.06.12.598645v1}{\underline{biorXiv}}.
(in revision, \textbf{Evolution})
\ifrpt 
\\[\pubspace em]
	\contribution{
		Collaboration with empirical research team.
		I contributed to writing, funding and idea development, and mentored on analyses.
		\\[\littlepubspace em]
	}
	\dohang
\fi
\end{longtable}
\unless\ifrpt
\pagebreak
\fi
%
\vspace{-1.5cm}
%
\begin{longtable}{>{\everypar{\dohang}\dohang\raggedright\arraybackslash}p{0.95\textwidth}}
\hfill\\
\rule{0pt}{3ex}\textbf{\underline{\smash{Published and Accepted}}}\\
\rule{0pt}{3ex}
%
%
30. \labbie{Grundler, MC}, J Terhorst, \bburd{GS Bradburd}.
\pubyear{2025}.
A geographic history of human genetic ancestry.
\journal{Science}.
387,1391-1397.
\\[\littlepubspace em]
\ifrpt 
	\contribution{
		This paper is led by a postdoc mentee in my lab (lead author). 
		I am senior author. 
		I contributed to writing and idea development, and mentored on analyses.
		\\[\littlepubspace em]
	}
	\dohang
\fi
%
29. \labbie{Clark, MI}, SW Fitzpatrick, \bburd{GS Bradburd}.
\pubyear{2024}.
Pitfalls and windfalls of detecting demographic declines using population genetics in long-lived species.
%\href{https://www.biorxiv.org/content/10.1101/2024.03.27.586886v1}{\underline{biorXiv}}.
\journal{Evolutionary Applications}.
17 (7), e13754.
\\[2 em]
\ifrpt 
	\contribution{
		%[\tinypubspace em]
		This paper is led by a PhD student in my lab (lead author). 
		I am senior author. 
		I contributed to writing and idea development, and mentored on analyses.
		\\[\tinypubspace em]
	} 
	\dohang
%\else 
%\\\pagebreak
\fi
%
28. Cocciardi JM, AM Hoffman, A Waananen, DL Des Marais, 
D Moeller, D Gamba, D Alvarado-Serrano, E Boehm, E Kottler, 
\bburd{G Bradburd}, H Branch, I Borokini, J Cavender-Bares, 
J Lau, J Anderson, J Jaros, K Toll, K Whitney, L Bolin, L Brudvig, 
M Ungerer, M Vahsen, M Blumstein, M Smith, M Howard, 
M Menon, NP Hanan, N Kooyers, R Shaw, S Sheth, S Wadgymar, 
T Mozdzer, T Juenger, T Chen, ML Avolio. 
\pubyear{2024}. 
The value of long-term ecological research for evolutionary insights. 
\journal{Nature Ecology \& Evolution}.
8, 1584–1592.
\\[-0.5 em]
\ifrpt 
	\contribution{
		\\[\tinypubspace em]
		Synthesis collaboration with national team of experts. 
%		Project coordinated via NSF Research Coordination Network
%		for which I am on the steering committee.
		I contributed to idea generation and writing.
		\\[\littlepubspace em]
	}
	\dohang
\else
\\ 
\fi
%
%
%\rule{0pt}{3ex}
27. \labbie{Hancock, Z}, \labbie{RH Toczydlowski}, \bburd{GS Bradburd}.
\pubyear{2024}.
A spatial approach to jointly estimate Wright's neighborhood size and long-term effective population size.
\journal{Genetics}.
227 (4).
\ifpm PMCID: PMC10029013 \fi
\\[-0.5 em]
\ifrpt 
	\contribution{
		\\[\tinypubspace em]
		This paper is led by a postdoc mentee in my lab (lead author). 
		I am senior author. 
		I contributed to writing and idea development, and mentored on analyses.
		\\[\littlepubspace em]
	} 
	\dohang
\else
\\
\fi
%
%
26. \labbie{Lewanski, AL}, \labbie{MC Grundler}, \bburd{GS Bradburd}.
\pubyear{2024}.
The era of the ARG: An introduction to ancestral recombination graphs and their significance in empirical evolutionary genomics.
\journal{PLoS Genetics}.
20(1): e1011110.
\ifpm PMCID: PMC10796009\fi
\\[\pubspace em]
\ifrpt 
	\contribution{
		\\
		This paper is led by a PhD student in my lab (lead author). 
		I am senior author. 
		I contributed to writing and idea development, and mentored on analyses.
		\\[\littlepubspace em]
	}
	\dohang
\fi
%
%
25. \labbie{Week, B} and \bburd{GS Bradburd}.
\pubyear{2024}.
%The era of the ARG: An introduction to ancestral recombination graphs and their significance in empirical evolutionary genomics.
Host-parasite coevolution in continuous space leads to variation in local adaptation across spatial scales.
\journal{American Naturalist}
203:1, 43-54.
\ifpm PMCID: PMC10796009 \fi
\\[\tinypubspace em]
\ifrpt 
	\contribution{
		\\
		This paper is led by a postdoc mentee in my lab (lead author). 
		I am senior author. 
		I contributed to writing and idea development, and mentored on analyses.
		\\[\tinypubspace em]
	} 
	\dohang
\fi
%
%
\rule{0pt}{3ex}
24. Mathur, S, A Mason, \bburd{GS Bradburd}, HL Gibbs.
\pubyear{2023}.
Functional genomic diversity is correlated with 
neutral genomic diversity in populations of an endangered rattlesnake.
\journal{Proceedings of the National Academy of Sciences}
120 (43).
\ifpm PMCID: PMC10614936 \fi
\\[\littlepubspace em]
\ifrpt 
	\contribution{
		\\
		Collaboration with empirical research team.
		I contributed to writing, funding and idea development, and mentored on analyses.
		\\[\tinypubspace em]
	} 
	\dohang
	\\\pagebreak
\else
\\[-\tinypubspace em]
\fi
%
%
\rule{0pt}{3ex}
23. Crandall, ED, \labbie{RH Toczydlowski}, L Liggins, 
AE Holmes, M Ghoojaei, MR Gaither, BE Wham, 
AL Pritt, C Noble, TJ Anderson, RL Barton, JT Berg, 
SG Beskid, A Delgado, E Farrell, N Himmelsbach, 
SR Queeno, T Trinh, C A Weyand, A Bentley, J Deck, 
C Riginos, \bburd{GS Bradburd}, RJ Toonen.
\pubyear{2023}.
The importance of timely metadata curation to the global surveillance of genetic diversity.
\journal{Conservation Biology} 37, e14061.
\ifpm PMCID: PMC10751740 \fi \\\\[-\tinypubspace em]
\ifrpt 
	\contribution{
		Collaboration with an international research team.
		I conceived of and executed statistical analyses, 
		and contributed to writing and idea development.
		\\[\tinypubspace em]
	} 
	\dohang
\fi 
%
%
\rule{0pt}{3ex}
22. Martin, BS, \bburd{GS Bradburd}, LJ Harmon, MG Weber.
\pubyear{2022}.
Modeling the evolution of rates of continuous trait evolution.
\journal{Systematic Biology} 72 (3), 590-605.
\ifpm PMCID: PMC10276627 \fi
\\\\[-0.78 em]
\ifrpt 
	\contribution{
		This paper is led by a PhD student (lead author) whose committee I am on. 
		I contributed to idea development and mentored on analyses.
		\\[\littlepubspace em]
	} 
	\dohang
\fi 
%
%
%\rule{0pt}{3ex}
21. \labbie{Clark, M}, \bburd{GS Bradburd}, M Akopyan, A Vega, E Rosenblum, J Robertson.
\pubyear{2022}.
Genetic isolation by distance underlies color pattern divergence in red-eyed treefrogs (\textit{Agalychnis callidryas}).
\journal{Molecular Ecology}. 31: 1666-1681.
\ifpm PMCID: PMC8923152 \fi 
\\\\[-0.7 em]
\ifrpt 
	\contribution{
		This paper was part of the Master's thesis of a student 
		who is currently a PhD student (lead author) in my lab. 
		I contributed to idea development and technical execution, 
		and mentored on analyses.
		\\[\tinypubspace em]
	} 
	\dohang
\fi 
%
%
\rule{0pt}{3ex}
20. \labbie{Toczydlowski, RH}, L Liggins, MR Gaither, TJ Anderson, RL Barton, 
JT Berg, SG Beskid, B Davis, A Delgado, E Farrell, M Ghoojaei, 
N Himmelsbach, AE Holmes, SR Queeno, T Trinh, CA Weyand, 
\bburd{GS Bradburd}, C Riginos, RJ Toonen, ED Crandall.
\pubyear{2021}.
Poor data stewardship will hinder global genetic diversity surveillance.
\journal{Proceedings of the National Academy of Sciences} 118 (34). 
\ifpm PMCID: PMC8403888 \fi
\\\\[-0.5 em]
\ifrpt 
	\contribution{
		This	collaboration with an international research team 
		is led by a postdoc mentee in my lab (lead author). 
		I contributed to idea development and writing.
		\\[\tinypubspace em]
	} 
	\dohang
\fi 
%
%
\rule{0pt}{3ex}
19. \labbie{Puckett, E}, S Murphy, \bburd{GS Bradburd}.
\pubyear{2021}.
Phylogeographic analysis delimits three evolutionary significant units of least chipmunks in North America and identifies unique genetic diversity within the imperiled Pe{\~{n}}asco population.
\journal{Ecology and Evolution} 11: 12114-12128.
\ifpm PMCID: PMC8427584 \fi
\\\\[-0.5 em]
\ifrpt 
	\contribution{
		This paper is led by a postdoc mentee in my lab (lead author). 
		I am senior author. 
		I contributed to writing and idea development 
		and executed several statistical analyses.
		\\[\littlepubspace em]
	} 
	\dohang
\fi 
%
%
18. Schweizer, RS, MR Jones, \bburd{GS Bradburd}, JF Storz, N Senner, C Wolf, ZA Cheviron. 
\pubyear{2021}.
Broad concordance in the spatial distribution of adaptive and neutral genetic variation along an elevational cline in deer mice.
\journal{Molecular Biology and Evolution} 38 (10), 4286-‌4300.
\ifpm PMCID: PMC8476156 \fi
\\\\[-0.5 em]
\ifrpt 
	\contribution{
		Collaboration with empirical research team.
		I contributed to writing and idea development, and mentored on analyses.
		\\[\tinypubspace em]
	} 
	\dohang
\fi 
%
%
17. \labbie{Hancock, ZB}, ES Lehmberg, \bburd{GS Bradburd}.
\pubyear{2021}.
Neo-darwinism still haunts evolutionary theory: A modern perspective on Charlesworth, Lande, and Slatkin (1982).
\journal{Evolution} 75: 1244-1255.
\ifpm PMCID: PMC8979413 \\ [\pubspace em]
\else \tabularnewline[\littlepubspace em]
\fi
\\\\[-1.8 em]
\ifrpt 
	\contribution{
		This paper is led by a postdoc mentee in my lab (lead author). 
		I am senior author. 
		I contributed to writing, scholarship, and idea development.
		\\[\tinypubspace em]
	} 
	\dohang
	\\\pagebreak
\fi 
%
%
16. Rothstein, A, R Knapp, \bburd{GS Bradburd}, D Boiano, EB Rosenblum.
\pubyear{2020}.
Stepping into the past to conserve the future: archived skin swabs from extant and extinct populations inform genetic management of an endangered amphibian.
\journal{Molecular Ecology} 29 (14): 2598-2611.
\ifpm  Research funded by NSF - no PMCID number. \tabularnewline  \\[-5pt]
\else \tabularnewline[\pubspace em]
\fi
\\\\[-1.5 em]
\ifrpt 
	\contribution{
		Collaboration with empirical research team.
		I contributed to writing and idea development, and mentored on analyses.
		\\[\tinypubspace em]
	} 
	\dohang
\else
\\[-\littlepubspace em]
\fi 
%
%
15. Fitzpatrick, SW \bburd{GS Bradburd}, CT Kremer, PE Salerno, LM Angeloni, WC Funk.
\pubyear{2020}.
Genomic and fitness consequences of genetic rescue in wild populations.
\journal{Current Biology} 30: 1-6.
\ifpm  Research funded by NSF - no PMCID number. \tabularnewline \\[-5pt]
\else  \\[\littlepubspace em] 
\fi
\\\\[-2.1 em]
\ifrpt 
	\contribution{
		Collaboration with empirical research team.
		I conceived of, and executed, a novel statistical approach 
		to analyze time-series sampling in admixed populations.
		I also contributed to idea development and writing. 
		This research led to a collaboration on a successful NSF proposal.
		\\[\tinypubspace em]
	} 
	\dohang
\fi 
%
%
14. Schweizer, RM, JP Velotta, CM Ivy, MR Jones, SM Muir, \bburd{GS Bradburd}, JF Storz, GR Scott, ZA Cheviron.
\pubyear{2019}.
Physiological and genomic evidence that a transcription factor contributes to adaptive cardiovascular function in high-altitude deer mice.
\journal{PLoS Genetics} 15 (11), e1008420.
\ifpm PMCID: PMC6837288 \fi
\\\\[-1.1 em]
\ifrpt 
	\contribution{
		\\
		Collaboration with empirical research team.
		I contributed to writing and idea development, and mentored on analyses.
		\\[\tinypubspace em]
	} 
	\dohang
\fi 
%
%
\rule{0pt}{3ex}
13. \bburd{Bradburd, GS} and PL Ralph.
\pubyear{2019}.
Spatial population genetics: It's about time. 
\hangindent1cm \journal{Annual Reviews in Ecology, Evolution, and Systematics} 50: 427-449.
\ifpm Research funded by NSF - no PMCID number. \fi
\\[\littlepubspace em]
\ifrpt 
	\contribution{
		\\[0.1 em]
		Invited synthesis paper. 
		I am lead author; 
		my co-author and I jointly generated ideas and co-wrote the manuscript.
		\\[\tinypubspace em]
	} 
	\dohang
\else 
\\[-\tinypubspace em]
\fi 
%
%
12. Grieneisen, LE, MJE Charpentier, SC Alberts, R Blekhman, \bburd{GS Bradburd}, J Tung, EA Archie.
\pubyear{2019}. 
Genes, geology, and germs: gut microbiota across a primate hybrid zone are explained by site soil properties, not host species.
\journal{Proceedings of the Royal Society B} 286: 20190431.
\ifpm PMCID: PMC6501927 \fi
\\\\[-1.1 em]
\ifrpt 
	\contribution{
	\\
		Collaboration with empirical research team.
		I conceived of, and executed, a novel statistical approach 
		to analyze gut microbial community composition as a function 
		of geography, diet, and host species.
		I also contributed to idea development and writing.
		\\[\tinypubspace em]
	} 
	\dohang
\else 
\\[-0.5 em]
\fi 
%
%
11. \rule{0pt}{1ex}\bburd{Bradburd, GS}, GM Coop, and PL Ralph.
\pubyear{2018}. 
Inferring continuous and discrete population genetic structure across space. 
\journal{Genetics} 210: 33-52.
\ifpm PMCID: PMC6116973 \fi
\\\\[-0.7 em]
\ifrpt 
	\contribution{
		I am lead author on this manuscript.
		I generated the research idea and plan, 
		developed the statistical method and associated software,
		analyzed the data, and wrote the manuscript.
		\\[\tinypubspace em]
	} 
	\dohang
\fi 
%
%
10. Weber, JN, \bburd{GS Bradburd}, YE Stuart, WE Stutz, DI Bolnick.
\pubyear{2017}. 
Partitioning the effects of isolation by distance, environment, and physical barriers on genomic divergence between parapatric threespine stickleback.
\journal{Evolution} 71: 342-56.
\ifpm Research funded by NSF - no PMCID number. \fi
\\\\[-0.5 em]
\ifrpt 
	\contribution{
		Collaboration with empirical research team.
		I contributed to writing and idea development, and mentored on analyses.
		\\[\tinypubspace em]
	} 
	\dohang
\fi 
%
%
9. Hoban, S, JL Kelley, KE Lotterhos, MF Antolin, \bburd{GS Bradburd}, DB Lowry, ML Poss, LK Reed, A Storfer, MC Whitlock.
\pubyear{2016}.
Finding the genomic basis of local adaptation in non-model organisms: pitfalls, practical solutions, and future directions. 
\journal{American Naturalist} 188: 379-397.
\ifpm PMCID: PMC5457800 \fi
\\\\[-0.5 em]
\ifrpt 
	\contribution{
		Synthesis paper with international research team. 
		I contributed to idea generation and writing.
		\\[\tinypubspace em]
	} 
	\dohang
	\\\pagebreak
\fi 
%
%
8. \rule{0pt}{1ex}\bburd{Bradburd, GS}, PL Ralph, GM Coop.
\pubyear{2016}. 
A spatial framework for understanding population structure and admixture. 
\journal{PLoS Genetics} 12: e1005703.
\ifpm PMCID: PMC4714911\fi
\\\\[-0.5 em]
\ifrpt 
	\contribution{
		I am lead author on this manuscript.
		I generated the research idea and plan, 
		developed the statistical method and associated software,
		analyzed the data, and wrote the manuscript.
		\\[\tinypubspace em]
	} 
	\dohang
\fi 
%
%
7. Hammock, BG, S Lesmeister, I Flores, \bburd{GS Bradburd}, FH Hammock, SJ Teh.
\pubyear{2016}. 
Low food availability narrows the tolerance of the copepod \textit{Eurytemora affinis} to salinity, but not to temperature. 
\journal{Estuaries and Coasts}.  39: 189-200.
\ifpm Research funded by NSF - no PMCID number. \fi
\\\\[-0.5 em]
\ifrpt 
	\contribution{
		Collaboration with empirical research team.
		I contributed to writing and mentored on analyses.
		\\[\tinypubspace em]
	} 
	\dohang
\fi 
%
%
6. Agrawal, AA, AP Hastings, \bburd{GS Bradburd}, EC Woods, T Z{\"u}st, J Harvey, T Bukovinszky.
\pubyear{2015}.
Evolution of plant growth and defense in a continental introduction. 
\journal{American Naturalist} 186: 1-15.
\ifpm Research funded by NSF - no PMCID number. \tabularnewline \\[-5pt]
\else \\[\littlepubspace em]
\fi
\\\\[-1.5 em]
\ifrpt 
	\contribution{
		Collaboration with empirical research team.
		I conceived of, and executed, a novel statistical approach 
		to analyze evolved plant defenses as a function 
		of relatedness and abiotic environmental factors.
		I also contributed to idea development and writing.
		\\[\tinypubspace em]
	} 
	\dohang
\else \\[-\littlepubspace em]
\fi 

%
5. Wang, IJ\dag and \bburd{GS Bradburd}\dag.
\pubyear{2014}. 
Isolation by environment. 
\journal{Molecular Ecology} 23: 5649-5662.
\newline(\dag co-first authors)
\ifpm Research funded by NSF - no PMCID number. \tabularnewline \\[-5pt]
\else \tabularnewline \\[-5pt]
\fi
\\\\[-2.4 em]
\ifrpt 
	\contribution{
		Synthesis paper. I am co-lead author; 
		my co-author and I jointly generated ideas and co-wrote the manuscript.
		\\[\tinypubspace em]
	} 
	\dohang
\fi 
%
%
4. \rule{0pt}{1ex}\bburd{Bradburd, GS}, PL Ralph, and GM Coop.
\pubyear{2013}. 
Disentangling the effects of geographic and ecological isolation on genetic differentiation. 
\journal{Evolution} 67: 3258-3273.
\ifpm PMCID: PMC3808528 \fi
\\[\littlepubspace em]
\ifrpt 
	\contribution{
		I am lead author on this manuscript.
		I generated the research idea and plan, 
		developed the statistical method and associated software,
		analyzed the data, and wrote the manuscript.
		\\[\tinypubspace em]
	} 
	\dohang
\fi 
%
%
3. Rejmanek, D, P Freycon, \bburd{GS Bradburd}, J Dinstell, and J Foley.
\pubyear{2013}.
Unique strains of \textit{Anaplasma phagocytophilum} segregate among diverse questing and non-questing \textit{Ixodes} tick species in the western United States.  
\journal{Ticks and Tick-borne Diseases} 4: 482-487.
\ifpm PMCID: PMC3833925 \fi
\\\\[-0.5 em]
\ifrpt 
	\contribution{
		Collaboration with empirical research team.
		I contributed to phylogenetic data analysis, 
		coding/figure generation, and writing/editing the manuscript.
		\\[\tinypubspace em]
	} 
	\dohang
\fi 
%
%
2. Rejmanek, D, \bburd{GS Bradburd}, and J Foley.
\pubyear{2012}.
Molecular characterization reveals distinct genospecies of \textit{Anaplasma phagocytophilum} from diverse North American hosts.	
\journal{Journal of Medical Microbiology} 61: 204-212.
\ifpm PMCID: PMC3352158 \fi
\\\\[-0.5 em]
\ifrpt 
	\contribution{
		Collaboration with empirical research team.
		I contributed to phylogenetic data analysis, 
		coding/figure generation, and writing/editing the manuscript.
		\\[\tinypubspace em]
	} 
	\dohang
\fi 
%
%
1. Near, TJ, CM Bossu, \bburd{GS Bradburd}, RL Carlson, RC Harrington, PR Hollingsworth Jr., BP Keck, DA Etnier.
\pubyear{2011}. 
Phylogeny and temporal diversification of darters (\textit{Percidae: Etheostomatinae}).  
\journal{Systematic Biology} 60: 565-595.
\ifpm Research funded by NSF - no PMCID number. \fi
\\\\[-0.5 em]
\ifrpt 
	\contribution{
		I contributed to data collection, data analysis, and editing the manuscript.
		\\[\littlepubspace em]
	} 
	\dohang
\fi
%
%
\unless\ifrpt
\pagebreak
\fi
%\end{etaremune}
%\hfill\\
\textit{\underline{\smash{Other}}} \hfill\\
\rule{0pt}{3ex}
Shaffer, HB, E McCartney-Melstad, PL Ralph, \bburd{GS Bradburd}, E Lundgren, J Vu, B Hagerty, F Sandmeier, C Weitzman, CR Tracy.
\pubyear{2017}. 
Desert Tortoises in the genomic age: population genetics and the landscape. 
\underline{bioRxiv} 
(Public comment on Desert Renewable Energy Conservation Plan).
\\[\littlepubspace em]
\ifrpt 
	\contribution{
		\\[\littlepubspace em]
		This paper is led by a postdoc mentee in my lab (lead author). 
		I am senior author. 
		I contributed to writing, idea development, and mentored on analyses.
		\\[\littlepubspace em]
	} 
	\dohang
\else 
\\
\fi 
\end{longtable}
%
\vspace{-1cm}
\section*{FUNDING}
\vspace{-0.6cm}
\rule{470pt}{0.4pt}
\begin{longtable}{>{\everypar{\dohang}\dohang\raggedright\arraybackslash}p{0.83\textwidth}p{0.15\textwidth}}
%\begin{tabular}{>{\everypar{\hangindent1cm}}p{0.8\textwidth}p{0.15\textwidth}}
%
\vspace{-0.6cm}
%

\textit{\underline{External}}\\
%\vspace{0.1cm}
\textbf{NSF Division of Environmental Biology} Award\# 2223962 (2022-2026) & \hfill \textbf{\$1,536,017}\\ 
\hspace{4.5mm} The interaction of selection, pleiotropy, and drift in phenotypic evolution\\
\hspace{4.5mm}(PI: J Conner; co-PIs: G Bradburd\ifpm \,(5\%)\fi, R Last, T Juenger)\\ 
\hspace{4.5mm}Bradburd lab budget: \$210,974.4\vspace{0.2cm}\\ 
%
\textbf{NIH NIGMS Early Stage Investigator MIRA (R35)} Award\# R35GM137919 (2020-2025) & \hfill \textbf{\$1,849,678} \\
\hspace{4.5mm} Incorporating geography into statistical methods for analysis of population genomic DNA\\
\hspace{4.5mm}(PI: G Bradburd\ifpm \,(50\%)\fi)\vspace{0.2cm}\\
%
\textbf{NSF Bridging Ecology \& Evolution} Award\# 2016569 (2020-2024) & \hfill \textbf{\$920,700} \\
\hspace{4.5mm} Testing eco-evolutionary effects of genetic drift and gene flow in stressful environments\\
\hspace{4.5mm}(PI: S Fitzpatrick; co-PIs: B Rothermel, G Bradburd\ifpm \,(5\%)\fi)\\
\hspace{4.5mm}Bradburd lab budget:\$157,590 \vspace{0.2cm}\\ 
%
\textbf{NSF Dimensions of Biodiversity} (2017-2022) & \hfill \textbf{\$1,118,520} \\
\hspace{4.5mm}Diversification of sensory systems in novel habitat\\
\hspace{4.5mm}(PI: J Boughman; co-PIs: G Bradburd, H Hofmann, J Keagy, D Stenkamp)\\ 
\hspace{4.5mm}Bradburd lab budget: \$158,851 \vspace{0.2cm}\\
%
\textbf{NSF Doctoral Dissertation Improvement Grant (DDIG)} (2014-15) & \hfill \textbf{\$13,918}\\
\hspace{4.5mm}The effect of intraspecific host variation on the structure of parasite populations\\
\hspace{4.5mm}(PI: G Bradburd; co-PI: G Coop)\\ \vspace{-0.1cm}

\textit{\underline{Internal}}\\\vspace{-0.25cm}

\textbf{MSU Strategic Partnership Grant} (2021-2022) & \hfill \textbf{\$400,000}\\
\hspace{4.5mm}The Institute for Biodiversity, Ecology, Evolution, and Macrosystems (IBEEM)\\
\hspace{4.5mm}(PI: PL Zarnetske. co-PIs: KS Cheruvelil, E Zipkin, KM Dahlin, G Bradburd, J Robinson)\\ \vspace{-0.1cm}

\textbf{Beacon - NSF Center for the Study of Evolution in Action} (2016-2021) & \hfill \textbf{\$127,579}\\
\hspace{4.5mm}Evolution of sensory systems in response to loss of visual information\\
\hspace{4.5mm}(PI: J Boughman, co-PIs: G Bradburd, H Hofmann, J Keagy, D Stenkamp)\\
%
\end{longtable}
\vspace{-0.3cm}
\section*{SOFTWARE}
\vspace{-0.6cm}
\rule{470pt}{0.4pt}
\begin{tabular}{>{\everypar{\hangindent1cm}}p{0.975\textwidth}}
\hfill\\
%
Grundler, M, J Terhorst, GS Bradburd (2024). \textbf{GAIA}. A method for inferring the geographic locations of 
shared genetic ancestors in a tree sequence. R package version v1.0.
\href{https://github.com/blueraleigh/gaia}{https://github.com/blueraleigh/gaia}\\
%
\vspace{-0.1cm}
%
Bradburd, GS (2018). \textbf{conStruct}. A genetic clustering method that accounts for isolation by distance. R package version v1.0.6.
\href{https://cran.r-project.org/web/packages/conStruct/index.html}{https://cran.r-project.org/web/packages/conStruct/index.html}\\
%
\vspace{-0.1cm}
%
Bradburd, GS (2016). \textbf{SpaceMix}. A program for estimating geogenetic maps from allele frequency data. R package version 0.13.
\href{https://github.com/gbradburd/SpaceMix}{https://github.com/gbradburd/SpaceMix}\\
%
\vspace{-0.1cm}
%
Bradburd, GS (2013). \textbf{BEDASSLE}. Bayesian Estimation of Differentiation in Alleles by Spatial Structure and Local Ecology. 
R package version 1.6.1. \href{https://cran.r-project.org/web/packages/BEDASSLE}{https://cran.r-project.org/web/packages/BEDASSLE}\\
%
\end{tabular}
%
\newpage
\section*{MENTORSHIP \& ADVISING}
\vspace{-0.6cm}
\rule{470pt}{0.4pt}
\hfill\\
\vspace{-0.9cm}
\begin{longtable}{>{\everypar{\hangindent1cm}}p{0.8\textwidth}p{0.15\textwidth}}
%
\textbf{\underline{Postdocs}}\\
\rule{0pt}{3ex}Seth Temple (Schmidt AI in Science Fellow) & \hfill 2024-present\\
Mike Grundler & \hfill 2022-present\\
Zach Hancock & \hfill 2021-present\\
Leonard Jones (NSF Postdoctoral Research Fellowship in Biology) & \hfill 2021-2024\\
\hspace{0.5cm} \textit{current position}: Bondermann Fellow, University of Washington \\
Phil Gerrish & \hfill 2022-2024\\
\hspace{0.5cm} \textit{current position}: Researcher, Los Alamos National Lab \\
Bob Week & \hfill 2020-2022\\
\hspace{0.5cm} \textit{current position}: Postdoctoral Researcher, University of Oregon \\
Rachel Toczydlowski & \hfill 2019-2022\\
\hspace{0.5cm} \textit{current position}: Research Landscape Geneticist, US Forest Service \\
Matteo Tomasini & \hfill 2019-2021\\
\hspace{0.5cm} \textit{current position}: Research Software Engineer, University of Gothenburg \\
Kelsey Yule & \hfill 2018-2019\\
\hspace{0.5cm} \textit{current position}: Project Manager of the NEON Biorepository, Arizona State University\\
Emily Puckett & \hfill 2017 - 2018\\
\hspace{0.5cm}\textit{current position}: Asst. Professor of Biological Sciences, U. Tennessee, Memphis\\
%
\\
%
\textbf{\underline{\smash{PhD Students}}}\\
\rule{0pt}{3ex}Alex Lewanski (DoD National Defense Science and Engineering Graduate Fellowship) & \hfill 2022-present\\
Meaghan Clark & \hfill 2019-2024\\
\hspace{0.5cm} \textit{current position}: NSF Postdoctoral Research Fellow, UC Santa Cruz\\
%
\\
%
\textbf{\underline{\smash{Master's Students}}}\\
\rule{0pt}{3ex}Ivana Barnes & \hfill 2024-present\\
%
\\
%
\textbf{\underline{\smash{Bioinformaticians/Technicians}}}\\
\rule{0pt}{3ex}Nicole Adams & \hfill 2022-present\\
Chris Talbot & \hfill 2024-present\\
\end{longtable}
%
\vspace{-0.5cm}
%
\begin{longtable}{>{\everypar{\hangindent1cm}}p{0.35\textwidth}p{0.6\textwidth}}
\textbf{\underline{Graduate student committee service}} \vspace{0.2cm}\\
\textit{\underline{\smash{Current graduate student committees}}} \vspace{0.2cm}\\
Mikayla Creek &\hfill Masters, Biology, James Madison University\\
Max Wytinski &\hfill PhD, EEB, University of Michigan\\
Nepsis Garc\'ia &\hfill PhD, EEB, University of Michigan\\
Eric Bastien &\hfill PhD, EEB, University of Michigan\\
Gayathri Venkatraman &\hfill PhD, EEB, University of Michigan\\
Kiran Kumar &\hfill PhD, Biostatistics, University of Michigan\\
Tristan Schramer &\hfill PhD, EEB, University of Michigan\\
Teresa Sauer &\hfill PhD, EEB, University of Michigan\\
Yu Kai Tan &\hfill PhD, EEB, University of Michigan\\
Sofia Maria Backlund &\hfill PhD, Institute of Science and Technology (Vienna)\\
Jimmy Bingman &\hfill PhD, Plant Biology, MSU\\
Matheus Januario Lopes de Sousa &\hfill PhD, EEB, University of Michigan \vspace{0.4cm}\\
%
\textit{\underline{\smash{Past graduate student committees}}} \vspace{0.2cm}\\
Bruce Martin &\hfill PhD 2024, Plant Biology, MSU\\
Enes Dilber &\hfill PhD 2023, Statistics, University of Michigan\\
Miranda Wade &\hfill PhD 2023, Integrative Biology, MSU\\
Sara Hugentobler &\hfill PhD 2023, Integrative Biology, MSU\\
Kyle Teller &\hfill PhD 2022, Integrated Applied Math, University of New Hampshire\\
Viviana Ortiz Londono  &\hfill  PhD 2022, Plant Pathology, MSU\\
Ava Garrison &\hfill PhD 2022, Plant Biology, MSU\\
%Devin Lake &\hfill PhD, Integrative Biology, MSU\\
%Jason Olsen &\hfill  PhD, Plant Biology, MSU\\
Michael Foisy &\hfill MSc 2020 , Plant Biology, MSU\\
Dr. Emily Dolson &\hfill PhD 2019, Computer Science and Engineering, MSU\\
%
\end{longtable}
%
\vspace{-0.5cm}
%
\begin{longtable}{>{\everypar{\hangindent1cm}}p{0.8\textwidth}p{0.15\textwidth}}
\textbf{\underline{\smash{Undergraduates}}}\\
\rule{0pt}{3ex}Ben Carlson & \hfill 2024 - present\\
Guoxiang (Jerry) Guo & \hfill 2024 - 2025\\
Lindsay Guare (MSU) & \hfill 2017 - 2019\\
Sarah Frocillo (MSU)  & \hfill 2017 - 2018\\
Nick Stants (MSU)  & \hfill 2017 - 2018\\
\end{longtable}
%
%\newpage
%\vspace{-1cm}
\section*{HONORS, AWARDS, \& FELLOWSHIPS}
\vspace{-0.6cm}
\rule{470pt}{0.4pt}
%
\begin{tabular}{>{\everypar{\hangindent1cm}}p{0.85\textwidth}p{0.1\textwidth}}
\hfill\\
\textbf{Fitch Award Finalist}, Society for Molecular Biology and Evolution (\$2000) & \hfill 2015\\
\textbf{Hamilton Award Finalist}, Society for the Study of Evolution (\$500) & \hfill 2015\\
\textbf{XSEDE SuperComputing Resource Allocation}, XSEDE (50,000 SUs) & \hfill 2014-05\\
\textbf{Center for Population Biology Research Fellowship} CPB, UC Davis (\$4,000) & \hfill 2011-13\\
%\textbf{Center for Population Biology Research Fellowship} CPB, UC Davis (\$2,000) & \hfill 2011-12\\
\textbf{NSF Graduate Research Fellow} National Science Foundation (\$90,000) & \hfill 2010-12\\
\textbf{Graduate Scholars Fellowship} in Population Biology, UC Davis (\$30,000) & \hfill  2009-10\\
%\hspace{4.5mm}Awarded in recognition of outstanding academic record and promise of productive scholarship\\
%\textbf{Environmental Internship Award} Yale University (\$6,000) & \hfill 2006-7\\ 
%\textbf{Richter Fellowship} Yale University (\$1,000)  & \hfill 2007\\
%\textbf{Mellon Forum Research Grant} Yale University (\$500) & \hfill 2007\\
%\textbf{Environmental Internship Award} Yale University (\$2,000) & \hfill 2006\\
\end{tabular}
%
\section*{TEACHING}
\vspace{-0.6cm}
\rule{470pt}{0.4pt}
%
\begin{tabular}{>{\everypar{\hangindent1cm}}p{0.60\textwidth}p{0.15\textwidth}p{0.175\textwidth}}
\hfill\\
\textbf{\underline{\smash{University of Michigan}}} \vspace{0.2cm}\\
Introduction to Statistical Model Building in R (EEB 429) & Instructor & \hfill WN23-25 \\
\multicolumn{3}{>{\raggedright\arraybackslash}p{\textwidth}}{\hspace{0.75cm}
\explain{Undergraduate course (~20-40 students/yr); introduction to coding and statistics in R.}} \\
Evolution (EEB 390) & Instructor & \hfill WN23,FA24 \\
\multicolumn{3}{>{\raggedright\arraybackslash}p{\textwidth}}{\hspace{0.75cm}
\explain{Undergraduate course (~100-150 students/yr); introduction to evolutionary biology.}} \\
\hfill\\
\textbf{\underline{\smash{Michigan State University}}} \vspace{0.2cm}\\
Evolutionary Biology (IBio 849) & Instructor & \hfill  SP18,19,20,22\\
\multicolumn{3}{>{\raggedright\arraybackslash}p{\textwidth}}{\hspace{0.75cm}
\explain{Graduate-level course (~20-40 students/yr); introduction to evolutionary biology.}} \\
Intro to Statistical Methods in Ecology/Evolution (IBio 830) & Instructor & \hfill FA17,18,19,20 \\
\multicolumn{3}{>{\raggedright\arraybackslash}p{\textwidth}}{\hspace{0.75cm}
\explain{Graduate-level course (~40-60 students/yr); introduction to coding and statistics in R.}} \\
Evolution Discussion Group (IBio 895) & Instructor & \hfill SP17,18,19,20 \\
\multicolumn{3}{>{\raggedright\arraybackslash}p{\textwidth}}{\hspace{0.75cm}
\explain{Graduate student-led discussion group on a rotating technical topic in evolutionary biology.}} \\
%UCLA/La Kretz Center Workshop in Conservation Genomics & Instructor & \hfill 2014, 16, 19 \\
%EVE100 Introduction to Evolution & TA & \hfill 2015	\\
%EVE103 Macroevolution, Speciation, and Phylogeny (Lecture, Lab) & TA & \hfill 2014 \\
%Herpetology (Field, Lab, Lecture) & TA & \hfill 2012\\
%Bodega Bay Applied Phylogenetics Workshop & TA & \hfill 2012-13\\
\end{tabular}

\ifrpt
\pagebreak
\fi

\section*{SERVICE \& LEADERSHIP}
\vspace{-0.6cm}
\rule{470pt}{0.4pt}
\begin{longtable}{>{\everypar{\dohang}\dohang\raggedright\arraybackslash}p{0.8\textwidth}p{0.15\textwidth}}
%\begin{tabular}{>{\everypar{\hangindent1cm}}p{0.8\textwidth}p{0.15\textwidth}}
\textit{\underline{\smash{National/International/Society Service}}}\\
\rule{0pt}{3ex}\textbf{Organizer} SMBE Symposium: "Spatial population genetics: where are we now?" 
%(co-organized with J. Novembre, UChicago) 
& \hfill 2024\\ 
%
\textbf{Organizing Committee} Midwest Population Genetics Conference & \hfill 2016-present\\
%
\textbf{\textit{Ad hoc}} \textbf{Associate Editor}, PLoS Genetics & \hfill 2025\\
%
\textbf{Conference Ally} Society for Molecular Biology and Evolution & \hfill 2024\\
%
\textbf{Steering Committee} NSF EVO-LTER Award: Leveraging long-term ecological research in grasslands: facilitating collaborations between ecologists and evolutionary biologists & \hfill 2021-present \\
\textbf{Course Instructor} UCLA/La Kretz Center Workshop in Conservation Genomics & \hfill 2014, 16, 19 \\
%
\textbf{Judge} Hamilton Award (Society for the Study of Evolution) & \hfill 2017, 19 \\
%
\textbf{Reviewer} including: 
PNAS;
Ecology Letters;
Evolution Letters;
Genetics;
Evolution;
American Naturalist;
Systematic Biology; 
Molecular Biology and Evolution;
Molecular Ecology;
Proceedings of the Royal Society B;
American Journal of Human Genetics;
%Heredity;
Molecular Ecology Resources;
Methods in Ecology and Evolution;
Molecular Phylogenetics and Evolution;
New Phytologist;
Bioinformatics;
Ecology\\[-0.2em]
%Ecological Monographs
%Systematics and Biodiversity;
%NSERC
%\textbf{Society Memberships}
%Society for the Study of Evolution (SSE); 
%Society for Molecular Biology and Evolution (SMBE); 
%American Society of Naturalists (ASN);
%Genetics Society of America (GSA)\\\\
%Diversity and Distributions;
%Integrative and Comparative Biology;
%Nature - Scientific Reports;
%Genes, Genomes, Genetics (G3);
%Ecology and Evolution;
%Australian National University
%
%
\\
\textit{\underline{\smash{Institutional Service}}}\\\\[-0.6em]
\textbf{\underline{\smash{University of Michigan}}} \\\\[-1.1em]
\rule{0pt}{3ex}Director, Frontiers Master's Program & \hfill 2025-present\\
%
Nominating Committee (College of LSA) & \hfill 2024\\
%
Executive Committee (EEB Dept.) & \hfill 2023-present\\
%
Executive Committee, UM Genome Science Training Program  & \hfill 2023-present\\
%
Collegiate Fellows Faculty Search Committee (EEB Dept.) & \hfill 2023\\
%
Founder, Exploring Careers Beyond Academia with Lunch (EEB Dept.) & \hfill 2023-present\\
%
\emph{Ad hoc} Faculty Search Committee (EEB Dept.) & \hfill 2023,2024\\
%
Seminar Committee (EEB Dept.)  & \hfill 2022-2023\\
%
\hfill\\
\textbf{\underline{\smash{Michigan State University}}} \vspace{0.2cm}\\
Diversity, Equity, and Inclusion Committee (Dept. Integrative Biology) & \hfill 2019-2022\\
%
Co-founder, Diversity, Equity, and Inclusion Reading Group (Dept. Integrative Biology) & \hfill 2019-2022\\
%
Co-chair, Presidential Postdoc Fellowship Committee (EEB Program) & \hfill 2020-2022\\
%
Seminar Committee (Dept. Integrative Biology) & \hfill 2020-2022\\
%
Chair Search Committee (Dept. Integrative Biology) & \hfill 2020-2022\\
%
Seminar Committee (EEB Program) & \hfill 2019-2020\\
%
Director Search Committee (EEB Program) & \hfill 2019-2020\\
%
\emph{Ad hoc} Space Committee (EEB Program) & \hfill 2019-2020\\
%
Strategic Hiring Planning Committee (Dept. Integrative Biology) & \hfill 2018-2019\\
%
%Curriculum Planning Committee (Intro to Evolution - IBio 445) & \hfill 2016-17\\
%
%Student Representative, Population Biology Grad Group Steering Committee & \hfill 2013-14\\
%
%Chair of Student Research Symposium, Center for Population Biology/Pop Bio Grad Group & \hfill 2013\\
%
%Course Organizer, The Bodega Bay Applied Phylogenetics Workshop & \hfill 2012-13\\
%
%Chair, UC Davis Non-Model Genomics Workshop Committee & \hfill 2010\\
%
%Graduate Student Mentor & \hfill 2009-2015\\
%
\end{longtable}
%
%
\section*{INVITED SEMINARS}
\vspace{-0.6cm}
\rule{470pt}{0.4pt}
%
\begin{longtable}{>{\everypar{\hangindent1cm}}p{0.4\textwidth}p{0.4\textwidth}p{0.125\textwidth}}
%
University of Michigan & University of Michigan Biological Station & \hfill 2024\\
%
European Society for Evolutionary Biology & Integration of Speciation Research & \hfill 2024\\
%
Louisiana State University & Dept. of Biological Sciences & \hfill 2022 \\
%
University of Michigan & Dept. of Ecology and Evolutionary Biology & \hfill 2022 \\
%
Institute of Science and Technology Austria & EvoLunch & \hfill 2021 \\
%
North Dakota State University & Dept.\ of Biological Sciences & \hfill 2021 \\
%
University of St. Andrews & Centre of Ecol. and Env. Modelling & \hfill 2021 \\
%
Cornell University & Dept.\ of Computational Biology & \hfill 2020 \\
%
Western Michigan University & Dept.\ of Biological Sciences & \hfill 2020 \\
%
Ohio State University & Dept.\ of Evolution, Ecology, Organismal Bio. & \hfill 2019 \\
%
University of Kentucky & Dept.\ of Biology & \hfill 2019 \\
%
Ecological Society of America & Landscape Genetics (student-organized) & \hfill 2018 \\
%
Indiana University & Dept.\ of Biology/Precision Health Initiative & \hfill 2018 \\
%
Penn State University & Ecology Seminar Series & \hfill 2018 \\
%
University of Toronto (St. George) & Dept.\ of Ecology and Evolutionary Biology & \hfill 2018 \\
%
University of Toronto (Mississauga) & Dept.\ of Biology & \hfill 2017 \\
%
University of Chicago & Dept.\ of Ecology and Evolution & \hfill 2017 \\
%
Michigan State University & Kellogg Biological Station & \hfill 2017 \\
%
University of Michigan & Center for Statistical Genetics & \hfill 2016 \\
%
University of Wyoming & Dept.\ of Botany & \hfill 2016 \\
%
University of Colorado Boulder & Dept.\ Ecology and Evolutionary Biology & \hfill 2016 \\
%
University of San Francisco & Dept.\ of Biology & \hfill 2016 \\
%
University of California Berkeley & Museum of Vertebrate Zoology & \hfill 2015 \\
%
Michigan State University & Ecology, Evolution, and Behavior & \hfill 2015 \\
%
Stanford University & Dept.\ Genetics and Biology & \hfill 2015 \\
%
University of Southern California & Molecular and Computational Biology & \hfill 2014 \\
%
University of Montana & Division of Biological Sciences & \hfill 2014 \\
%
University of Minnesota & Dept.\ of Plant Biology & \hfill 2014 \\
%
Sonoma State University & Biology Dept.\ & \hfill 2014 \\
%
University of Texas, Austin & Dept.\ Integrative Biology & \hfill 2013
%
\end{longtable}
%
\section*{CONTRIBUTED PRESENTATIONS}
\vspace{-0.6cm}
\rule{470pt}{0.4pt}
\begin{tabular}{>{\everypar{\hangindent1cm}}p{0.75\textwidth}p{0.2\textwidth}}
\hfill\\
Society for the Study of Evolution & \hfill 2012-13,15,17,19,23\\
Population, Evolutionary, Quantitative Genetics Meeting & \hfill 2018, 2022\\
Midwestern Population Genetics Meeting & \hfill 2016\\
American Society of Naturalists & \hfill 2016,18\\
Society for Molecular Biology and Evolution & \hfill 2015\\
%International Association of Landscape Ecology & \hfill 2015\\
Bay Area Population Genomics (BAPG) & \hfill 2014\\
%CPB Dept. Seminar & \hfill 2011,13\\
%E\&EB Senior Symposium, Yale University & \hfill 2008\\
\end{tabular}

\end{document}

%%%%%%%%%%%%%%
%%%%%%%%%%%%%%
%%%%%%%%%%%%%%
%	GRAVEYARD
%%%%%%%%%%%%%%
%%%%%%%%%%%%%%
%%%%%%%%%%%%%%

\section*{PREVIOUS APPOINTMENTS}
\vspace{-0.6cm}
\rule{470pt}{0.4pt}
\begin{tabular}{>{\everypar{\hangindent1cm}}p{0.8\textwidth}p{0.15\textwidth}}
\hfill\\
\textbf{Research Assistant, Prof. Thomas Near, Yale University} & \hfill 2008-09 \\ 
\hspace{4.5mm}Carried out research on phylogeography of North American fishes & \\
%
\textbf{Technician, Yale Peabody Museum of Natural History} & \hfill 2007-08 \\ 
\hspace{4.5mm}Identified and sorted vertebrate specimens&\\ 
%
\textbf{Lab Technician, Marine Biology Lab, Prof. Leo Buss} & \hfill 2005 \\ 
\hspace{4.5mm}Maintained populations of \textit{Hydractinia echinata} and \textit{Nematostella vectensis}&
\end{tabular}

\section*{FIELDWORK}
\vspace{-0.6cm}
\rule{470pt}{0.4pt}
\begin{tabular}{>{\everypar{\hangindent1cm}}p{0.8\textwidth}p{0.15\textwidth}}
\hfill\\
\textbf{Abaco Islands (Bahamas)} & \hfill 2013\\
\hspace{4.5mm}Arthropod surveys and sampling; Anole sampling.\\
%
\textbf{Northern California} & \hfill 2010-2012\\
\hspace{4.5mm}Sampled tissue and ectoparasites from lizards and small rodents.\\
%
\textbf{American Southeast and Midwest} & \hfill 2008\\ 
\hspace{4.5mm}Sampled freshwater fish throughout the Mississippi Drainage.\\
%
\textbf{Costa Rica}  & \hfill  2007\\ 
\hspace{4.5mm}Collected reptiles and amphibians in the rainforest for the Yale Peabody Museum.\\ 
\hspace{4.5mm}Developed skills in collecting and preserving herpetological specimens.\\
\hspace{4.5mm}Monitored status of the Variable Harlequin Toad (\textit{Atelopus varius}).\\
%
\textbf{Suriname}  & \hfill  2006\\
\hspace{4.5mm}Collected birds in coastal and southern Suriname for the Yale Peabody Museum.\\ 
\hspace{4.5mm}Developed skills in collecting (mist-netting, shooting) and preserving ornithological specimens (anatomicals, skeletons, skins).\\ 
\end{tabular}


\section{\underline{LANGUAGES}}
\textbf{Spanish}: proficient speaking and reading\\
\textbf{Sranan Tongo (Taki-taki)}: fair, but only in the context of collecting specimens in the field

\textbf{Relevant Coursework}
\begin{myindentpar}{6mm}
{UC Davis: Core Principles in Population Biology, Mathematical Methods in Population Biology, Macroevolution, Principles of Biological Data Analysis}\vspace{3mm}

Yale: General Chemistry (w/ lab), General Physics, General Ecology, Evolutionary Biology, Diversity of Life, Molecular Systematics (intensive lab), Genetics, Conservation Genetics, Phylogenetic Methods and Approaches, Biology of Terrestrial Arthropods (w/ lab), Ornithology (w/ lab), Introductory Geoscience, Petrology and Mineralogy, Natural Resources and Sustainability
\end{myindentpar}

\textbf{Integrative Biology Fellowship} UT Austin, 2009-'10 (declined)\\ \vspace{0.3mm}
\hspace{4.5mm}Awarded to top applicants in the UT system\\
\textbf{Dorothy Clark Lettice Fellowship} Kansas University, 2009-'10 (declined)\\  \vspace{0.3mm}
\hspace{4.5mm}Awarded in recognition of high academic ability\\

\section{\underline{VOLUNTEER AND LEADERSHIP EXPERIENCE}}
\textbf{Yale Freshman Outdoor Orientation Trips (FOOT) Leader} 2005-'07\\ \vspace{0.3mm}
\hspace{4.5mm}Led 4-6 day trips in the wilderness with 8-10 incoming Yale freshmen\\ \vspace{0.3mm}
\hspace{4.5mm}Trained and certified in Wilderness First Aid and CPR\\
\textbf{FOOT Public Relations Core Head} 2006-'07\\ \vspace{0.3mm}
\hspace{4.5mm}Organized all public relations initiatives for the FOOT program, including sponsorship bids, new\\ \vspace{0.3mm} 
\hspace{4.5mm}leader recruitment, and advertising to incoming freshmen\\ 
\textbf{Volunteer Income Tax Assistance Site Coordinator, New Haven} 2005-'07\\ \vspace{0.3mm}
\hspace{4.5mm}Coordinated an office and trained volunteers to administer Earned Income Tax Credit to local\\ \vspace{0.3mm}
\hspace{4.5mm}individuals and families\\ 
\textbf{Local Director of Client Services, National Student Partnerships, New Haven} 2004-'06\\ \vspace{0.3mm}
\hspace{4.5mm}Worked with socioeconomically disadvantaged clients to navigate social bureaucracy and find\\ \vspace{0.3mm}
\hspace{4.5mm}employment, affordable housing, and public benefits\\ \vspace{0.3mm}
\textbf{Community Health Educator, New Haven} 2005-'06\\ \vspace{0.3mm}
\hspace{4.5mm}Taught in local schools to provide health education, specifically in communication skills with\\ \vspace{0.3mm}
\hspace{4.5mm}reference to substance and relationship abuse, nutrition, and sexual health\\
\textbf{DEMOS Classroom Volunteer, Lincoln-Bassett Elementary School, New Haven} 2004-'05\\ \vspace{0.3mm}
\hspace{4.5mm}Demonstrated basic scientific principles in a fun and accessible way\\ 
\textbf{Bradburd, G.S.}, P.L. Ralph, G.M. Coop.   Disentangling the effects of geographic and ecological isolation on genetic differentiation.  Society for the Study of Evolution. 2013. Snowbird, Utah.\

\textbf{Bradburd, G.S.}  P.L. Ralph, B.Moore, G.M. Coop. A Bayesian method for estimating genetic differentiation due to isolation by geographic and ecological distance.  Society for the Study of Evolution. 2013. Ottawa, Canada.\

\textbf{Bradburd, G.S.} Retrieving lost evolutionary history from an extinct darter species.  Ecology and Evolutionary Biology Senior Symposium, Yale University, 2008.

\textbf{Bradburd, G.S.}, B.Moore, G.M. Coop, J.E. Foley. Molecular ecology of two tick-borne pathogens in California.  Center for Population Biology Departmental Seminar. 2011. UC Davis.\

\section*{INVITED TALKS}
\vspace{-0.6cm}
\rule{470pt}{0.4pt}
%
\begin{tabular}{>{\everypar{\hangindent1cm}}p{0.85\textwidth}p{0.1\textwidth}}
\hfill\\
%
\textbf{Bradburd, G.S.} Patterns of Spatial Genetic Variation. & \hfill 2015 \\
\hspace{1cm} Michigan State University.\\
%
\textbf{Bradburd, G.S.}, P. Ralph, G. Coop. The Geography of Genetic Structure and Admixture. & \hfill 2015 \\
\hspace{1cm} Stanford University.\\
%
\textbf{Bradburd, G.S.}, P. Ralph, G. Coop. A new approach to the geography of genetic structure. & \hfill 2014 \\
\hspace{1cm} University of Southern California.\\
%
\textbf{Bradburd, G.S.}, P. Ralph, G. Coop. The spatial context of genetic admixture. & \hfill 2014 \\
\hspace{1cm} University of Montana.\\
%
\textbf{Bradburd, G.S.}, P. Ralph, G. Coop.  A spatial framework for studying genetic differentiation.  & \hfill 2014 \\
\hspace{1cm} University of Minnesota.\\
%
\textbf{Bradburd, G.S.} The effects of geographic and ecological isolation on genetic differentiation. & \hfill 2014 \\
\hspace{1cm} Sonoma State University. \\
%
\textbf{Bradburd, G.S.} \textit{BEDASSLING} your data: background and functions of the R package \textit{BEDASSLE}.  & \hfill 2013 \\
\hspace{1cm} University of Texas, Austin\\
\end{tabular}